Makefile.3ds

```diff
@@ -176,9 +176,13 @@ endif
 $(OUTPUT).3dsx: $(OUTPUT).elf
 $(OUTPUT).elf:  $(OFILES)
 
-#$(OUTPUT).smdh	:	$(APP_ICON)
-#	@bannertool makesmdh -s "$(APP_TITLE)" -l "$(APP_DESCRIPTION)"  -p "$(APP_AUTHOR)" -i $(APP_ICON) -o $@
-#	@echo "built ... $(notdir $@)"
+$(OUTPUT).smdh	:	$(APP_ICON)
+	@bannertool makesmdh -s "$(APP_TITLE)" -l "$(APP_DESCRIPTION)"  -p "$(APP_AUTHOR)" -i $(APP_ICON) -o $@
+	@echo "built ... $(notdir $@)"
+
+$(TARGET).bnr	:	$(TOPDIR)/$(BNR_IMAGE) $(TOPDIR)/$(BNR_AUDIO)
+	@bannertool	makebanner -o $@ -i $(TOPDIR)/$(BNR_IMAGE) -a $(TOPDIR)/$(BNR_AUDIO)
+	@echo "built ... $@"
```

Comment:
I'm sorry, I intended to only have the change in the Makefile not the Makefile.3ds. I was not aware it would also grab the Makefile.3ds which I committed after making the pull request. Sorry!